tools/Microsoft.AspNet.Server.Kestrel.GeneratedCode/KnownHeaders.cs

```diff
@@ -59,7 +59,7 @@ protected string Term(string name, int offset, int count, string array, string s
             {
                 ulong mask = 0;
                 ulong comp = 0;
-                for (var scan = 0; scan != count; ++scan)
+                for (var scan = 0; scan < count; scan++)
```

Comment:
This is the generator.

@lodejard uses some some unconventional looping styles sometimes. I hadn't considered this would break JITing heuristics. I think that's a good enough reason to accept this PR.